idrstream/download.py

```diff
@@ -118,11 +118,10 @@ def download_image(
         """
         save_dir.mkdir(parents=True, exist_ok=True)
         image_path = self.get_IDR_mitocheck_movie_path(plate, well_num)
-        idr_location = f"{self.idr_id}@fasp.ebi.ac.uk:{image_path} "
+        idr_location = f"fasp-public@fasp.ebi.ac.uk:/pub/databases/IDR/{self.idr_index_name }/{image_path} "
```

Comment:
```suggestion
        idr_location = f"fasp-public@fasp.ebi.ac.uk:/pub/databases/IDR/{self.idr_index_name}/{image_path} "
```
Noticed space at the end of `self.idr_index_name`, so I recommend removing it to keep formatting the same.

Reply:
Oops, good catch!